Adding the onnxruntime Dockerfile and instructions (#968)
microsoft/onnxruntime: dockerfiles/README.md

  docker run -it onnxruntime-ngraph
  ```

## ONNX Runtime Server (Preview)
#### Linux 16.04

1. Build the docker image from the Dockerfile in this repository
  ```

>   docker build -t {docker_image_name} .

Reviewer: ```suggestion
  docker build -t {docker_image_name} -f Dockerfile.server .
```

  ```
  

> 2. Run the ONNXRuntime server in Docker image

Reviewer 2: ```suggestion
2. Run the ONNXRuntime server with the image created in step 1
```

  ```

>   docker run -it -v {localModelAbsoluteFolder}:{dockerModelAbsoluteFolder} -e MODEL_ABSOLUTE_PATH={dockerModelAbsolutePath} -p {your_local_port}:8001 {imageName}

Reviewer 3: why is -it needed?

Author: Removed

  ```

> 3. Send the request to server

Reviewer 2: ```suggestion
3. Send an HTTP request to the container running ONNX Runtime Server
```
